ee/tabby-db/src/lib.rs

```diff
@@ -49,6 +49,54 @@ pub struct DbConn {
     cache: Arc<DbCache>,
 }
 
+fn make_pagination_query_with_condition(
```

Comment:
consider move code that will be used by macros into a separate file, e.g `src/macro_deps.rs`

Reply:
It's not only used by macros for the time being, so I think we should leave it here until the refactoring is complete.